<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDialogLogin_salary</class>
 <widget class="QDialog" name="UDialogLogin_salary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вхід до програми &quot;Ель-Зар&quot;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_pidpryemstva">
     <property name="title">
      <string>Підприємства</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboBox_pidpryemstva"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_password">
     <property name="title">
      <string>Пароль</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_clearPswrd">
        <property name="text">
         <string>Очистити</string>
        </property>
        <property name="icon">
         <iconset resource="../uPictRc.qrc">
          <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/editclear-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/editclear-256.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_accept">
       <property name="text">
        <string>Увійти</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/agt_action_success-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/agt_action_success-256.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_reject">
       <property name="text">
        <string>Відмінити</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/button_cancel-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/button_cancel-256.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_pidpryemstva</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton_accept</tabstop>
  <tabstop>pushButton_reject</tabstop>
 </tabstops>
 <resources>
  <include location="../uPictRc.qrc"/>
 </resources>
 <connections/>
</ui>
